plumbing/object/tag_test.go

```diff
@@ -234,41 +234,40 @@ func (s *TagSuite) TestString(c *C) {
 	)
 }
 
-func (s *TagSuite) TestTagToTagString(c *C) {
+func (s *TagSuite) TestStringNonCommit(c *C) {
```

Comment:
I was fixing the test because after changing the Write to auto-count the size, I figure out that the test was wrong.

1) The test functions are called "Test<method>[variant]" 
2) The test was wrong
3) Didn't worked after autocalculate the size, because you can't use a custom hash in the objects. But since size was 0, the hash was not being calculated.